### PR TITLE
update documentation more details for users installs

### PR DESCRIPTION
**Pull request by walejandromt**
### Summary
I just add the example file for installation and update Readme.md for more detail of the file "value.yaml" of the helm

### Type of Change
This PR fixes/implements the following **bugs/features**:
- Feature 2 -> update documentation more details for users installs

### How has this been tested?
Proof:
No need tested only change documentation

### Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly.
- [ ] I have written new tests for my changes.
  - [x] My changes successfully ran and pass tests locally.

### CODE REVIEW

File: examples/config-install/values.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+operator.image: "zufardhiyaulhaq/ngrok-operator"
```
Comment (zufardhiyaulhaq) [inline examples/config-install/values.yaml:1]: I don't think we need this, the example values is already in `charts/ngrok-operator/values.yaml`

File: README.md [hunk 2]
```diff
@@ -35,6 +35,7 @@ https://ngrok.zufardhiyaulhaq.com
 ```
 
 ## Values
+see example files [here](examples/config-install/values.yaml)
```
Comment (zufardhiyaulhaq) [inline README.md:38]: I don't think another file is needed, can you change it to `charts/ngrok-operator/values.yaml` instead?

Comment (zufardhiyaulhaq) [reply README.md:38]: also can you update the README in `charts/ngrok-operator/README.md.gotmpl`? thanks

**Comment by zufardhiyaulhaq** [comment]
Thanks @walejandromt for contributing!
add some comment related the PR.

File: examples/config-install/values.yaml (hunk 1 above)
Comment (walejandromt) [reply examples/config-install/values.yaml:1]: ok sorry  i dont not viewed this file i delete this file new

File: README.md (hunk 2 above)
Comment (walejandromt) [reply README.md]: yes no need another file, i change to `charts/ngrok-operator/values.yaml` and update `charts/ngrok-operator/README.md.gotmpl`, then is it necesary update charts/ngrok-operator/README.md` too?

**Pull request merged**